README.md

```diff
@@ -45,6 +45,103 @@ Required installed programs: Bash and jq for bash, Python3 and the PyVCF, filelo
+
+This version of the output results contains one row for each sample/study pair. It also includes columns listing the rsIDs of the snps involved in the risk score calculation. 
+
+Sample | Study ID | Reported Trait | Trait | Citation | P-Value Annotation | Beta Annotation | Score Type | Units (if applicable) | SNP Overlap | SNPs Excluded Due To Cutoffs | Included SNPs | Used Super Population | Polygenic Risk Score | Protective Variants | Risk Variants | Variants Without Risk Allele | Variants in High LD
+
+.. code-block:: bash
+
+   ./runPrsCLI.sh -f path/to/file/samples.vcf -o path/to/file/output.tsv -c 0.0005 -r hg19 -p SAS -v
+
+### JSON
+
+This version outputs the results in a json object format. The output automatically contains all the data the full version does and there is no condensed version of the json output. The file will contain a list of json study objects. Each study object will contain the list of samples and their score. 
+
+.. code-block:: bash
+    # example output
+    [
+        {
+            "studyID": "GCST001",
+            "reportedTrait": "Alzheimer's Disease",
+            "trait": "Alzheimer Disease",
+            "citation": "First Author et al. 2021",
+            "pValueAnnotation": "NA",
+            "betaAnnotation": "NA",
+            "scoreType": "OR",
+            "units (if applicable)": "NA",
+            "snpOverlap": 8,
+            "excludedSnps": 3,
+            "includedSnps": 14,
+            "usedSuperPop": "EUR",
+            "samples": [
+                {
+                    "sample": "SAMP001",
+                    "polygenicRiskScore": "1.277",
+                    "protectiveAlleles": "rs1|rs2|rs3",
+                    "riskAlleles": "rs4|rs5|rs6|rs7|rs8",
+                    "variantsWithoutRiskAllele": "rs9|rs10|rs11|rs14",
+                    "variantsInHighLD": "rs12|rs13"
+                },
+                {
+                    "sample": "SAMP002",
+                    "polygenicRiskScore": "NF",
+                    "protectiveAlleles": "",
+                    "riskAlleles": "",
+                    "variantsWithoutRiskAllele": "rs1|rs2|rs3|rs4|rs5|rs6|rs7|rs8|rs9|rs10|rs11|rs14",
+                    "variantsInHighLD": "rs12|rs13"
+                },
+                {
+                    "sample": "SAMP003",
+                    "polygenicRiskScore": "1.63",
+                    "protectiveAlleles": "rs1|rs2|rs3",
+                    "riskAlleles": "rs4|rs5|rs6|rs7|rs8",
+                    "variantsWithoutRiskAllele": "rs1|rs2|rs3|rs4|rs5|rs6|rs7|rs8|rs9|rs10|rs11|rs14",
+                    "variantsInHighLD": "rs12|rs13"
+                }
+            ]
+        }
+    ]
+
+.. code-block:: bash
+
+   ./runPrsCLI.sh -f path/to/file/samples.vcf -o path/to/file/output.json -c 0.0005 -r hg19 -p SAS
+
 ## Citing this work
 
 Please visit the [Studies](https://prs.byu.edu/studies.htm) page of the website for information on how to cite this tool and GWAS publications used. 
```

Comment:
##License

This work is freely available for academic and not-for-profit use. However, commercial use is regulated by © 2020 Brigham Young University. All rights reserved. For more information about commercial use of this product, please contact Justin Miller, Ph.D. (justin.miller@uky.edu) or Keoni Kauwe, Ph.D. (kauwe@byu.edu).

Reply:
Is this instead of the Citing this work section or in addition to?

Reply:
In addition too
```suggestion
Please visit the [Studies](https://prs.byu.edu/studies.htm) page of the website for information on how to cite this tool and GWAS publications used. 

##License

This work is freely available for academic and not-for-profit use. However, commercial use is regulated by © 2020 Brigham Young University. All rights reserved. For more information about commercial use of this product, please contact Justin Miller, Ph.D. ([justin.miller@uky.edu](mailto:justin.miller@uky.edu)) or Keoni Kauwe, Ph.D. ([kauwe@byu.edu](mailto:kauwe@byu.edu)).
```